### DISCUSSION TITLE
Welcome to Weight-Tracker Discussions!

### DISCUSSION THREAD

**Discussion by alicolville** [yekenuk/Weight-Loss-Tracker#324] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by YOPT21** [comment]
As there are no discussions here, I thought I'd start one..
Hey everyone, my name is Si and I am the owner of YourOnlinePersonalTrainer and I use both the Pro Weight Tracker and Meal Tracker on my website which is also being developed into an App. I look forward to hearing about your projects and how you are using these projects. Hopefully we can all help each other out.

**Comment by mybestrong** [comment]
How have I only just seen this!! :)

Im Rick, one half of Be Strong Eat Move Live. We're a Community Interest Company based in Lancashire UK since Jan 2016.

We've developed so many resources, had a brilliant run right up to Covid and its been very challenging since.

Really interested in connecting with UK/Ireland organisations that use a weekly lifestyle/weight loss group model.

Regards,

Rick

**Reply by YOPT21** [reply]
Hi Rick! Good to hear from you. I love the name 👍 sounds amazing. Covid has knock a fair few of us back unfortunately, but its set backs like this that make us stronger! I am based in North Devon and I help my clients reach their fitness potential with the useage of Boxercise. I am currently using Ali's plugins  to track my clients progress and to help them see their own progress. How are you finding the plugins yourself?

Simon 
![Screenshot_20220822-163930_Your Online Personal Trainer](https://user-images.githubusercontent.com/82499071/185961842-214f496e-8e94-4a4b-8868-23d75ccb5551.jpg)
![Screenshot_20220822-163936_Your Online Personal Trainer](https://user-images.githubusercontent.com/82499071/185961848-34e2c69e-f59b-4b35-bf9a-b677190e4e10.jpg)
![Screenshot_20220822-163943_Your Online Personal Trainer](https://user-images.githubusercontent.com/82499071/185961855-ff59edb4-1e84-476f-95c8-2b96dc4e2028.jpg)

**Comment by mybestrong** [comment]
Hi Simon,

 

We work on habit transformation and have developed relationships with phycologists at 2 major universities who we collaborate with. 

 

In terms of weight/meal tracker we’ve been with Ali since the early days and I like to think we’ve contributed a fair bit. Ali will tell you we’ve just been a pain in the butt!! 😉

 

In a nutshell we love the plugins and we’re very loyal to Ali – he’s been awesome with us.

 

What’s working best for you in terms of recruiting new members at the moment? We used to do so well from Facebook but now, unless you’re paying for ads, there is very little traction. The irony is that they wont allow us to run ads as they class us in one of the categories (weight loss) that isn’t allowed. Interestingly Noom and Slimming World seem to get away with it but I guess that’s the money talking!

 

Best wishes,

 

Rick

 

 

 

Rick Wilson

 

Be  Strong

Eat. Move. Live CIC

 

www.mybestrong.com

**Reply by YOPT21** [reply]
Thats incredible! Changing people habbits will definitely improve their health for a life time rather than a quick fix. You seem to have a great business model and a very high success rate. 

I've been using the plugin for awhile now and have thrown a few ideas over to Ali and he has been amazing with the responses and updates through the plugin. I wouldn't go anywhere else to be honest. 

Alot of my clients come from word of mouth and my loyalty scheme. I have a loyalty card that stores on the clients phone via google wallet or the apple wallet, they recieve a stamp per booking and then recieve a reward once they book X amount of sessions. They also have a 'share' button that allows them to share with friends. If a friend then books they both recieve a stamp. I am super lucky that I have a brother that is an online marketing expert and gives me a lot of his time to help my business.

**Reply by mybestrong** [reply]
That's great news re your bro!! We did employ somebody full time in social media marketing but we still didn't see any tangible benefits. We're working now around 3 groups on Tuesday, Wednesday and Thursday evenings and are likely to revert to the old fashioned flyers, posters and physical handouts of vouchers etc in the localities.

**Reply by YOPT21** [reply]
The way social media is going, sometimes the old fashioned way is the best. If you are on the lookout for prints for your flyers etc the  can I recommend www.devonpixels.co.uk
